### DISCUSSION TITLE
Is window object supported?

### DISCUSSION THREAD

**Discussion by maguroid** [fwouts/previewjs#400] [Q&A]
Hi. I wanna use window object (specifically, using window.solana object).
Is that case supported?
Thank you for your help.

**Comment by fwouts** [comment]
Hi @maguroid! You can run any initialisation code you like inside your Wrapper component: https://previewjs.com/docs/config/wrapper. You could probably require that library from there :)

Let me know how you go!

**Reply by maguroid** [reply]
@fwouts 
Thank you for your replying !
I understand. But window.solana object is [injected by Phantom browser extension](https://docs.phantom.app/integrating/detecting-the-provider). So I need to define provider manually [like this](https://github.com/solana-labs/browser-extension/blob/e7797f35f714118af88ee5ce2d51cb11b4e88d64/app/inpage/inpage.ts).
It's little bit tough...

**Reply by fwouts** [reply]
That is a very unusual use case unfortunately!

**Reply by maguroid** [reply]
That's right. I gonna simply check on browser so far...
Thanks!
